scripts/install/configuration/replace_ssh_keys.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+
+usage()
+{
+   # Display Help
+   echo "Replaces the ssh keys in Kubernetes"
+   echo "NOTE: This does not update deployed keys"
+   echo
+   echo "Usage: replace_ssh_keys.sh [ --public-key-file file ]"
+   echo "                            [ --private-key-file file ]"
+   echo
+   echo "Options:"
+   echo "public-key-file      File path for a public key."
+   echo "private-key-file     File path for a private key."
+   echo
+}
+
+if [[ $# -eq 0 ]]; then
```

Comment:
The usage statement indicates that the arguments are optional (the square brackets imply that). Looking at the script, it looks like you want one or both to be specified. I don't know if there is a clean way to show that just with symbology, short of writing multiple usage lines. I would add a line at the end of the usage message saying "At least one option must be specified."

Reply:
I added the comment.  I know I had them as optional, but I'm not sure how to say "at least one of the following" in argument syntax.